docs/documentation/providers.md

```diff
@@ -197,6 +197,7 @@ Datafaker comes with a total of 233 data providers:
 | [Pokemon](https://javadoc.io/doc/net.datafaker/datafaker/latest/net/datafaker/providers/entertainment/Pokemon.html)                                             |                                                                                                                                                                                                             | Entertainment | 0.8.0 |
 | [Princess Bride](https://javadoc.io/doc/net.datafaker/datafaker/latest/net/datafaker/providers/entertainment/PrincessBride.html)                                |                                                                                                                                                                                                             | Entertainment | 0.8.0 |
 | [Programming Language](https://javadoc.io/doc/net.datafaker/datafaker/latest/net/datafaker/providers/base/ProgrammingLanguage.html)                             |                                                                                                                                                                                                             | Base          | 0.8.0 |
+| [Pronouns](https://javadoc.io/doc/net.datafaker/datafaker/latest/net/datafaker/providers/base/Pronouns.html)                                                    | Gender pronouns are the set of pronouns that an individual uses to reflect that person's own gender identity                                                                                                | Base          | 2.1.0 |
```

Comment:
This shouldn't be modified this is part of the release tasks (IIRC)

Reply:
Correct, but it doesn't hurt.

Reply:
That's one of the points about release. The code, doc, and version will be there without the artifact generated and publicly available.

I can follow any recommendation.